code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -196,39 +196,39 @@
 			filtered_out.temperature = removed.temperature
 
 			if(scrub_Toxins && removed_gases["plasma"])
-				filtered_out.assert_gas("plasma")
+				filtered_out.add_gas("plasma")
```

Comment:
add_gas can only be used instead of assert_gas if you're 100% sure that gas doesn't already exist in the gas_mixture. Are you 100% sure?

Reply:
`filtered_out` gets new'd a few line above: `var/datum/gas_mixture/filtered_out = new`

So I'm 100% sure